repositories/repositories.bzl

```diff
@@ -79,26 +79,41 @@ def repositories():
             name = "containerregistry",
             sha256 = "a8cdf2452323e0fefa4edb01c08b2ec438c9fa3192bc9f408b89287598c12abc",
             strip_prefix = "containerregistry-" + CONTAINERREGISTRY_RELEASE[1:],
+            # containerregistry assumes a repository named @httplib2, but
+            # https://github.com/bazelbuild/bazel/issues/3998 means that this
+            # causes problems with the import path. Instead, we use
+            # repo_mapping and a fully-qualified repository name.
```

Comment:
I'm not sure that using a repo mapping will avoid name conflicts. Last I looked, repo mapping affected how bazel processed labels, but not what name it used for the repo underneath a target's runfiles root. And the runfiles root is one of the things put on `PYTHONPATH`.

Reply:
Renaming the repository from "httplib2" to "com_github_httplib2_httplib2" resolves the name conflict, but breaks because containerregistry expects the name "httplib2". Would it make it clearer if I move the comment to the definition of @httplib2, and change it to:

```
# Other repositories, including @containerregistry, expect this be called @httplib2. However,
# due to #3998, this causes naming conflicts, so we use the fully qualified name here.
```

Reply:
I see, your rename works not because of the repo_mapping per se but because the http_archive.name attribute was also changed. Makes sense.